sdk/cognitiveservices/Vision.Face/src/Generated/IPersonDirectory.cs

```diff
@@ -258,8 +275,16 @@ public partial interface IPersonDirectory
         /// <param name='persistedFaceId'>
         /// PersistedFaceId created from Person Face Create
         /// </param>
-        /// <param name='body'>
-        /// Target person face id to update.
+        /// <param name='detectionModel'>
+        /// Possible values include: 'detection_01', 'detection_02',
+        /// 'detection_03', 'detection_preview_1904', 'ir_detection_01',
+        /// 'expression_01'
+        /// </param>
+        /// <param name='faceFeature'>
```

Comment:
probably need docs, what is face feature?

Reply:
That field was intentionally left blank to not touch the scope. The field is effectively useless as the backend does not do anything with it and will not do anything with it, but it is in the contract and we have not gotten to the service side refactoring. 
For this reason, the corresponding argument is manually removed in the updated method in PersonDirectoryExtension that uses the flattened body. The field is just there with the old method for now, waiting for when we can properly fix this with a somewhat breaking change starting from the service side.

Reply:
[nit] remove xmldoc of removed parameters